.gulp/regeneration.iced

```diff
@@ -1,21 +1,26 @@
-
 ###############################################
 # LEGACY 
 # Instead: have bunch of configuration files sitting in a well-known spot, discover them, feed them to AutoRest, done.
 
+rimraf = require "rimraf"
```

Comment:
Should I just say OK, this works, I'm leaving it alone?

Reply:
That's not a complaint, by the way

Reply:
No, you should understand what it is. rimraf is a npm package that allows me to delete files and folders easily. I use it on line 34 to delete the contents of the folder at the baseFolderPath.